packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -86,19 +95,42 @@ function FlowsheetDataGrid(props) {
     rowHeight,
     onCellSelect,
     onClearSelectedCells,
+    intl,
   } = props;
 
+  // Replace each non-header cell that contains no content with a dash indicating "No results".
+  const parsedRows = rows.map(
+    (row) => ({
+      ...row,
+      cells: row.cells.map(
+        (cell, index) => ((!cell.content && index !== 0) ? {
+          ...cell,
+          content: (
+            <>
+              <span aria-hidden>{intl.formatMessage({ id: 'Terra.flowsheetDataGrid.no-result-display' })}</span>
+              <VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.flowsheetDataGrid.no-result' })} />
+            </>
+          ),
+        } : cell),
+      ),
+    }),
+  );
+
+  // Make all columns not resizable.
+  const nonResizablePinnedColumns = columns.length ? [{ ...columns[0], isResizable: false }] : [];
+  const nonResizableOverflowColumns = columns.length > 1 ? columns.slice(1).map(column => ({ ...column, isResizable: false })) : [];
```

Comment:
Maybe I am not thinking properly here, but couldn't we just map all the columns the same way in one call and then separate as needed for the props?

Reply:
Right, that makes more sense to do the operation all at once and then dividing it. Done.